kinder/pkg/constants/constants.go

```diff
@@ -73,6 +73,10 @@ const (
 	// TODO: send a PR to define this value in a kind constant (currently it is not)
 	KubeadmConfigPath = "/kind/kubeadm.conf"
 
+	// KubeadmIgnorePreflightErrorsFlag holds the default list of preflight errors to skip
+	// on "kubeadm init" and "kubeadm join"
+	KubeadmIgnorePreflightErrorsFlag = "--ignore-preflight-errors=Swap,SystemVerification,ImagePull,FileContent--proc-sys-net-bridge-bridge-nf-call-iptables"
```

Comment:
with `ci/latest` the `ImagePull` check fails.
this PR already merged:
https://github.com/kubernetes/kubernetes/pull/85965/files

so i'm going to assume that we were ignoring these errors before too.

ideally we should not ignore this preflight check.
possibly, eventually making kubeadm support `ImagePullPolicy` as per the discussion with ncdc.

/hold